### PR TITLE
Feature branch: Home page and About page

### PR DESCRIPTION
**Pull request by nouman17610**
I implemented the following tasks

- Home page responsive design
- About page responsive design
- Hamburger menu on mobile screen
- Dynamic speakers posts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,142 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please support the project by adding your contributions.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my coding partner Collins and Filber who discussed the technical areas with me.
+
```
Comment (laiifuu) [inline README.md]: As requested in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.), you should give proper credit to the author behind the original design, [Cindy Shin](https://www.behance.net/adagio07).

File: README.md [hunk 2]
```diff
@@ -1 +1,142 @@
...
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
```
Comment (laiifuu) [inline README.md:16]: As mentionned in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Deploy%20the%20project%20and%20check%20to%20make%20sure%20everything%20works%20as%20planned.), you should deploy your project using `GitHub Pages`. Once you do that, add the link to your deployed project to your `README`.

**Review by laiifuu — CHANGES_REQUESTED** [review]
## Hi @nouman17610 :wave: 

Good job so far!
There are some issues that you still need to work on but you are almost there!

cat-driving-serious


## Highlights
- [x] Green linters check :heavy_check_mark: 
- [x] Descriptive PR summary and title :heavy_check_mark: 
- [x] Great commit messages :heavy_check_mark: 
- [x] Speakers section dynamically loaded with JS :heavy_check_mark: 
- [x] Nice use of semantic elements :+1: 

## Required Changes ♻️
- [ ] Nice work on the header :+1: It would be great if you could add some spacing to adjust the nav container in mobile view because the burger menu and the logo are stuck to the edges of the screen: 
    - Your design: 
    image
    - The template: 
    image

- [ ] Nice work done on the header :+1: However there are some issues you still need to work on: 
    - The text in the black bar at the top is supposed to be white,
    - The `Campaign` button text is supposed to be orange,
    - There's an `x` that's appearing in the middle of the white navigation bar that should be removed 
        - You design: 
        image
         - The template: 
         image

- [ ] Nice work done on the `Main Topics` sec tion :+1: However there are a few things you need to adjust: 
    - There should be a small orange bar dividing the title `Main Topics` and the topics cards, 
    - There should be a hover effect when hovering over the topics cards, that turns the border of the cards to white, 
    - The text `SEE THE WHOLE PROGRAM` should be underlined, 
    - There should be more space at the top and bottom of the section,
        - Your design: 
        image
        - The template: 
        image

- [ ] In mobile view for the `Main Topics` section: 
    - There should be more spacing between the topics cards, 
    - `See the whole program` should be replaced with an orange button, 
        - Your design: 
        image
        - The template: 
        image

- [ ] In the main speakers section, there should be a small orange bar separating the title of the section, and the speakers: 
    - Your design: 
    image
    - The template: 
    image

- [ ] In the main speakers section, the image for the first speaker doesn't appear: 
image

- [ ] For the first section in the `About` page, there should be a line of text above the big title in mobile view: 
    - Your design: 
    image

    - The template: 
    image

- [ ] There's an issue with the width of the page since i can horizontally scroll the page: 
image
image

- [ ] As mentionned in the requirements, you are supposed to personalize the content of your page based on the theme you chose, so for the the second section of the `About` page, you should change its content but keep the skeleton. You should also add a small orange bar beneath the title of the section to match the design:
Your design: 
image

- [ ] For the third section, you should also personalize the content of the title and the text beneath it, and add a small orange bar under the section's title: 
Your design: 
image

In the mobile view, the images are taking a larger space then they should, kindly fix this issue: 
    - Your design: 
    image
    - The template: 
    image

- [ ] Nice job using semantic tags in your web pages :+1:  It would be even better if you could use the semantic `article` tag for your topic boxes in the `Home` page.




_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by laiifuu** [comment]
I'm sorry i accidentally submitted the review before finishing it, please wait before going ahead with making changes :pray:

**Comment by nouman17610** [comment]
> I'm sorry i accidentally submitted the review before finishing it, please wait before going ahead with making changes 🙏

@laiifuu 
Okay, no problem. Please in your final changes, identify the selector that needs to display none to remove the cross button from the desktop version.

**Comment by laiifuu** [comment]
> > I'm sorry i accidentally submitted the review before finishing it, please wait before going ahead with making changes pray
> 
> @laiifuu Okay, no problem. Please in your final changes, identify the selector that needs to display none to remove the cross button from the desktop version.

It's this element: 
image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,730 @@
...
+}
+
+.topic {
+  background-color: #282b32;
+  text-align: center;
+}
+
+.topic-area {
+  padding-top: 35px;
+}
```
Comment (AbelG101) [inline style.css:169]: - [ ] As mentioned previously please make sure to add some vertical spacing between the cards in the "Main Topics" section in the mobile version as shown in the Behance design.
- Your design:
  image
- Behance design:

File: style.css [hunk 4]
```diff
@@ -0,0 +1,730 @@
...
+  visibility: hidden;
+}
+
+.banner-area {
+  background-image: url('images/banner.png');
+  background-repeat: no-repeat;
+  background-position: absolute;
+  background-size: cover;
+  width: 100%;
+}
```
Comment (AbelG101) [inline style.css:111]: - [ ] As mentioned previously the width in the About page is overflowing and creating a scroll behavior so I'd advise you fix it as it leads to a bad UX.
- Here's how it looks:
  FtIBjCsD1R

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @nouman17610 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Content is uniquely tailored ✔
- UI is looking good ✔
- Correct use of GitHub Flow ✔
- Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @nouman17610,

✔️ No overflows and nice design

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**